charts/lws/templates/prometheus/monitor.yaml

```diff
@@ -14,8 +14,7 @@ spec:
       path: /metrics
       port: https
       scheme: https
-      tlsConfig:
-        insecureSkipVerify: true
+      tlsConfig: {{- toYaml .Values.metrics.serviceMonitor.tlsConfig | nindent 6 }}
   selector:
     matchLabels:
```

Comment:
does the prometheus collection in the helm deployment work for you?
IMO `controller-manager-metrics-service` should have this label to work properly

Reply:
You are right. Updated.

Reply:
I think this still applies as there is no service updated in the commits